scripts/oncotree_to_oncotree.py

```diff
@@ -467,49 +476,63 @@ def write_summary_file(target_file, source_version, target_version):
     # For each category, codes with more granular codes introduced are shown first
     unmappable_codes = sorted([unmappable_code for unmappable_code, unmappable_node in GLOBAL_LOG_MAP.items() if unmappable_node[NEIGHBORS_FIELD]])
     ambiguous_codes = sorted([ambiguous_code for ambiguous_code, ambiguous_node in GLOBAL_LOG_MAP.items() if len(ambiguous_node[CHOICES_FIELD]) > 1], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
-    resolved_codes = sorted([resolved_code for resolved_code, resolved_node in GLOBAL_LOG_MAP.items() if len(resolved_node[CHOICES_FIELD]) == 1], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
+    resolved_codes = sorted([resolved_code for resolved_code, resolved_node in GLOBAL_LOG_MAP.items() if len(resolved_node[CHOICES_FIELD]) == 1 and not ("???" in resolved_node[CHOICES_FIELD]) ], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
+    unrecognized_codes = sorted([unrecognized_code for unrecognized_code, unrecognized_node in GLOBAL_LOG_MAP.items() if len(unrecognized_node[CHOICES_FIELD]) == 1 and ("???" in unrecognized_node[CHOICES_FIELD]) ], key = lambda k: sort_by_resolution_method(k, GLOBAL_LOG_MAP[k]))
 
     html_summary_file = os.path.splitext(target_file)[0] + "_summary.html"
     with open(html_summary_file, "w") as f:
         # General info
         f.write("<!DOCTYPE html>\n<html lang=\"en\">\n<head>\n<title>Mapping Summary</title>\n<meta charset=\"UTF-8\">\n<style>\nbody {font-family:Arial; line-height:1.4}\n\n</style>\n</head><body>\n")
         f.write("<h1>Mapping Summary</h1>\n")
-        #f.write("<p><b>Source Version</b>: %s<br />\n<b>Target Version</b>: <a href=\"%s\">%s</a> *<i>All resolutions should be made with this version</i><br /><br />\n" % (source_version, oncotree_url, target_version))
-        f.write("<p>Mapped <b>%s</b> to <b>%s</b><br />" % (source_version, target_version))
+        #f.write("<p><b>Source Version</b>: %s<br>\n<b>Target Version</b>: <a href=\"%s\">%s</a> *<i>All resolutions should be made with this version</i><br><br>\n" % (source_version, oncotree_url, target_version))
```

Comment:
can we remove comments?

Reply:
oops .. missed this one. We can clean comments in follow up PR next time we are refining